translations/ui-agreements/en_US.json

```diff
@@ -455,7 +455,7 @@
     "settings.displaySettings": "Display settings",
     "settings.general.displaySuppressFromDiscovery.description": "Display \"Suppress from discovery\" setting on",
     "settings.general.displaySuppressFromDiscoveryAgreementLine.title": "Agreement lines",
-    "settings.general.displaySuppressFromDiscoveryPCI.title": "Package content items",
+    "settings.general.displaySuppressFromDiscoveryPCI.title": "Titles in packages",
```

Comment:
Now that you are adding new translations, we dont need to add entries to the `en_US` file here.